src/routes/profile/index.tsx

```diff
@@ -28,6 +28,7 @@ import { getAvatarLibrarySizeForUser } from '../../rewards/getRewards';
 
 const MAX_ACTIVE_AVATAR_REQUESTS = 1;
 const MIN_GAMES_REQUIRED = 100;
+const MIN_GAMES_REQUIRED_FOR_TOURNEY = 20; // This should be less than MIN_GAMES_REQUIRED
```

Comment:
Chuck an assert on this so the app dies on startup if this constraint isn't satisfied.

Reply:
`assert(MIN_GAMES_REQUIRED_FOR_TOURNEY < MIN_GAMES_REQUIRED)`